### DISCUSSION TITLE
Xiaomi 11t "App not installed as package appears to be invalid."

### DISCUSSION THREAD

**Discussion by realmkoy** [MlgmXyysd/Xiaomi-HyperOS-BootLoader-Bypass#218] [Q&A]
When i try to downgrade my settings app it just says "App not installed as package appears to be invalid." on the xiaomi 11t
I tried the settings app on 123pan but they dont work and the one in github link. please help ps. miui was installed previously but i went to a mobile shop to fix the camera and they installed hyperos for no reason

**Comment by Mostwest** [comment]
Anything that was shipped with Miui as default, should be able to replace the settings.apk with one method or the other
